### PR TITLE
vscode-extensions.formulahendry.terminal: init at 0.0.10

### PR DESCRIPTION
**Pull request by VergeDX**
###### Things done
- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/misc/vscode-extensions/default.nix [hunk 1]
```diff
@@ -646,39 +646,53 @@ let
...
+
+        terminal = buildVscodeMarketplaceExtension {
+          mktplcRef = {
+            publisher = "formulahendry";
+            name = "terminal";
+            version = "0.0.10";
+            sha256 = "sha256-9hGkD/mWGhwH0ACA3nUD75/XCIi6A8DiDVagfHwPRz4=";
+          };
+          meta = {
+            license = lib.licenses.mit;
```
Comment (figsoda) [inline pkgs/misc/vscode-extensions/default.nix:694]: I can't find a license in the repository or on vs marketplace, are you sure this is mit licensed?

Comment (VergeDX) [reply pkgs/misc/vscode-extensions/default.nix:694]: I cannot found them too... But I noticed these issues:
https://github.com/formulahendry/vscode-terminal/issues/32
https://github.com/formulahendry/vscode-auto-rename-tag/issues/553
and assume the license is mit...

**Comment by bobby285271** [comment]
The extension has no updates for 4 years and the author says `formulahendry.code-runner` should be a superset of this extension [[1]](https://github.com/formulahendry/vscode-terminal), do we still need this?

**Comment by VergeDX** [comment]
No.

**Pull request closed without merging**